### ISSUE TITLE
I've made a working one

### ISSUE THREAD

**Issue by LanLan69** [rawandahmad698/PyChatGPT#134]
https://github.com/LanLan69/easyChatGPT

**Comment by sajjadGG** [comment]
put a __init__ file in classes folder and in the main.py instead of `.classes` use `classes`

**Comment by aaronteamAAR** [comment]
> https://github.com/LanLan69/easyChatGPT

it does'nt let you use chatgpt from the terminal

**Comment by aetherwu** [comment]
It dose work but it lanuches a browser evey time you run it...

**Comment by aaronteamAAR** [comment]
> It dose work but it lanuches a browser evey time you run it...

Yes that's the issue, please is there a way I can maintain a live communication?

**Comment by v1s1t0r999** [comment]
> > It dose work but it lanuches a browser evey time you run it...
> 
> Yes that's the issue, please is there a way I can maintain a live communication?

There's an argument `--headless` (or smth iirc). It doesn't launch any browser. The argument is for selenium btw.
